DNN Platform/Website/Templates/Default Website.template.en-US.resx

```diff
@@ -296,7 +296,7 @@ Fax: (555) 555-4564</value>
     <value>Extensions</value>
   </data>
   <data name="Tab.Home.CompanyIntro.Text" xml:space="preserve">
-    <value>DNN (formerly DotNetNuke) provides a suite of solutions for creating rich, rewarding online experiences for customers, partners and employees.  DNN products and technology are the foundation for 750,000+ websites worldwide.  In addition to our commercial CMS and Social Community solutions, DNN is the steward of the DotNetNuke Open Source Project.</value>
+    <value>DNN (formerly DotNetNuke) provides a suite of solutions for creating rich, rewarding online experiences for customers, partners and employees.  DNN products and technology are the foundation for 750,000+ websites worldwide.  In addition to our commercial CMS and Social Community solutions, DNN is the steward of the DNN Open Source Project.</value>
```

Comment:
Is this verbiage correct?  I think the end of this should be changed to DNN Corporation is the ....  Otherwise it seems to read incorrectly

Reply:
question would be: do we still advertise Evoq products inside DNN Platform?

Reply:
@mitchelsellers: You are correct. I did not change the original text, that text was already wrong in the first place.
@sleupold:  I shall change the line and remove "In addition [...] ,"

I will create a new PR that corrects this individual file or, if you reject this PR, I can resubmit with this file corrected as wel.